impyute/dataset/base.py

```diff
@@ -65,6 +70,48 @@ def randn(theta=(0, 1), shape=(5, 5), missingness="mcar", thr=0.2, dtype="float"
     raw_data = getattr(corruptor, missingness)()
     return raw_data
 
+def randc(nlevels=5, shape=(5, 5), missingness="mcar", thr=0.2):
+    """ Return randomly generated dataset with uniformly distributed categorical data (alphabetic character)
+
+    Parameters
+    ----------
+    :param nlevels: int
```

Comment:
Slight changes need to be done to the way the documentation is formatted. Sphinx is used for the documentation and [`autodoc`](http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) is used to automatically generate the documentation using docstrings... And [`numpydoc` ](https://numpydoc.readthedocs.io/en/latest/format.html) is the format used by `autodoc`. Could you please format according to `numpydoc`? Here's the link https://numpydoc.readthedocs.io/en/latest/format.html but the tl;dr is to remove the `:param` stuff and add a
```
Returns
-------
None
```

Reply:
huh sorry, missed this part, will do!